ctapipe/calib/camera/r1.py

```diff
@@ -110,6 +104,19 @@ def check_r0_exists(self, event, telid):
                 self._r0_empty_warn = True
             return False
 
+    @classmethod
+    def from_eventsource(cls, eventsource=None, *args, **kwargs):
```

Comment:
maybe call it `for_eventsource`? I know that `from_` is very widespread for alternative constructors, but semantically, we want a given calibrator **for** our eventsource

Reply:
sounds good to me.